imap/utils.go

```diff
@@ -16,7 +18,18 @@ func login(ctx context.Context, d *plugin.QueryData) (*client.Client, error) {
 	port := 993
 	tlsEnabled := true
 	insecureSkipVerify := false
-	var host, login, password string
+
+	// Check env var settings
+	host := os.Getenv("IMAP_HOST")
```

Comment:
@graza-io Can you please update `config/imap.spc` and `docs/index.md` mentioning these env variables? For `config/imap.spc`, can you please add a line like https://github.com/turbot/steampipe-plugin-aws/blob/ed9e5a8b2b463fb6e30bd2fc19bbb16ab9dc8607/config/aws.spc#L34? And for `docs/index.md`, can you please update the `Configuration` section like in https://hub.steampipe.io/plugins/turbot/aws#configuration?

Reply:
https://github.com/turbot/steampipe-plugin-imap/pull/18/commits/bf44bca7c39bf223105638347785be78046421ea